lib/shopify_app/configuration.rb

```diff
@@ -26,6 +28,14 @@ def has_webhooks?
     def has_scripttags?
       scripttags.present?
     end
+
+    def has_fulfillment_services?
+      fulfillment_services.present?
+    end
+
+    def has_base_url?
+      base_url.present?
+    end
```

Comment:
I have gone down this route of base_url before and I found it got kind of complicated. I kind of prefer to have people set the url explicitly in the initializer